test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4059,9 +4059,13 @@ def _log_match(self, logtype, msg, id=None, n=50, tail=True,
             rv = self.logutils.match_msg(lines, msg, allmatch=allmatch,
                                          regexp=regexp, starttime=starttime,
                                          endtime=endtime)
-            if rv is None and not existence:
-                self.logger.log(level, infomsg + attemptmsg + '... OK')
-                break
+            if not existence:
+                if rv:
+                    _msg = infomsg + ' - Match'
```

Comment:
Just minor nit: Instead `- Match` how about `- but exists` or `- exists`, so it will be more meaningful like `log match: searching for "msg" - with non-existence - but exists`

Reply:
@hirenvadalia done the changes.